src/components/Organisms/AssetTile/AssetTile.tsx

```diff
@@ -56,6 +59,7 @@ export const AssetTile: React.FC<AssetTileProps> = ({
 
   const history = useHistory()
   const ref = useRef(null)
+  const assetDuration = `${asset.duration}${' '}${asset.durationType}`
```

Comment:
The AssetMeta component could do this.
```suggestion
  const assetDuration = `${asset.duration}${' '}${asset.durationType}`
```